### PR TITLE
feat(hugging face): add hugging face connector

### PR DESCRIPTION
**Pull request by igpetrov**
feat(hugging face): add hugging face connector

### CODE REVIEW

File: connectors/hugging-face/element-templates/hugging-face-connector.json [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+      },
+      "constraints": {
+        "notEmpty": true
+      }
+    },
+    {
+      "id": "inputType",
+      "label": "Input type",
+      "group": "input",
+      "description": "Whether to use simplified or fine-tuned input",
```
Comment (bastiankoerber) [inline connectors/hugging-face/element-templates/hugging-face-connector.json:88]: we should link here the https://huggingface.co/docs/api-inference/detailed_parameters
and rename fine-tuning with advamced input configuration or sth. similar

**Review by bastiankoerber — APPROVED** [review]

File: connectors/hugging-face/element-templates/hugging-face-connector.json [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+      },
+      "constraints": {
+        "notEmpty": true
+      }
+    },
+    {
+      "label": "Model",
+      "group": "input",
+      "feel": "optional",
+      "value": "gpt2",
```
[the remark this answers is not in the document]
Comment (igpetrov) [reply connectors/hugging-face/element-templates/hugging-face-connector.json:73]: I am prefilling values to allow a quick-start for a user. If you want, I can remove that.

File: connectors/hugging-face/element-templates/hugging-face-connector.json [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+      "binding": {
+        "type": "zeebe:input",
+        "name": "inputType"
+      }
+    },
+    {
+      "label": "Input",
+      "group": "input",
+      "feel": "optional",
+      "value": "Camunda is a great choice for my business because",
```
[the remark this answers is not in the document]
Comment (igpetrov) [reply connectors/hugging-face/element-templates/hugging-face-connector.json:110]: Okay

File: connectors/hugging-face/element-templates/hugging-face-connector.json [hunk 4]
```diff
@@ -0,0 +1,225 @@
...
+      "condition": {
+        "property": "inputType",
+        "equals": "simplified"
+      }
+    },
+    {
+      "label": "Input",
+      "group": "input",
+      "feel": "required",
+      "description": "Desired input for your model",
```
[the remark this answers is not in the document]
Comment (igpetrov) [reply connectors/hugging-face/element-templates/hugging-face-connector.json:129]: I thought, that would be covered in the official docs. But I can add a reference here too.

File: connectors/hugging-face/element-templates/hugging-face-connector.json (hunk 1 above)
Comment (igpetrov) [reply connectors/hugging-face/element-templates/hugging-face-connector.json]: Removed dropdown for now as discussed offline

File: connectors/hugging-face/element-templates/hugging-face-connector.json (hunk 4 above)
[the remark this answers is not in the document]
Comment (igpetrov) [reply connectors/hugging-face/element-templates/hugging-face-connector.json]: Added a link now

File: connectors/hugging-face/element-templates/hugging-face-connector.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (igpetrov) [reply connectors/hugging-face/element-templates/hugging-face-connector.json]: Removed for now. Let me know if we should revert

File: connectors/hugging-face/element-templates/hugging-face-connector.json [hunk 5]
```diff
@@ -0,0 +1,225 @@
...
...12.687 77.9572 112.801C76.7971 112.916 75.6267 112.742 74.55 112.295C73.6893 111.94 72.9072 111.418 72.2488 110.759C71.5903 110.101 71.0684 109.319 70.713 108.458C70.267 107.381 70.0935 106.211 70.2082 105.051C70.3228 103.891 70.7219 102.777 71.37 101.808C72.1488 100.642 73.2558 99.7333 74.5512 99.1967C75.8466 98.6601 77.272 98.5197 78.6471 98.7935C80.0223 99.0672 81.2853 99.7427 82.2764 100.734C83.2675 101.726 83.9422 102.99 84.215 104.365C84.4883 105.74 84.3477 107.165 83.8113 108.46C83.2748 109.755 82.3654 110.861 81.2 111.64ZM182.613 111.64C181.644 112.288 180.53 112.687 179.37 112.801C178.209 112.916 177.039 112.742 175.962 112.295C175.101 111.939 174.319 111.418 173.661 110.759C173.003 110.101 172.481 109.319 172.125 108.458C171.68 107.381 171.507 106.211 171.621 105.051C171.736 103.891 172.135 102.777 172.782 101.808C173.364 100.936 174.133 100.205 175.032 99.6658C175.931 99.1269 176.938 98.7942 177.981 98.6917C179.025 98.5891 180.078 98.7193 181.064 99.0728C182.051 99.4264 182.947 99.9944 183.688 100.736C184.68 101.727 185.355 102.99 185.628 104.365C185.902 105.74 185.761 107.165 185.224 108.46C184.687 109.755 183.779 110.861 182.613 111.64Z' fill='%23FFAD03'/%3E%3C/svg%3E"
```
[the remark this answers is not in the document]
Comment (igpetrov) [reply connectors/hugging-face/element-templates/hugging-face-connector.json]: This one is an official asset. Found smaller one.

**Pull request merged**